docs/slack.rst

```diff
@@ -268,9 +268,8 @@ In addition to the guidance that applies to all participants, we expect self-pro
 Here are some channels specifically for wholesome self-promotion:
 
 - ``#job-posts-only`` and ``#freelance`` are the best places to post paid jobs and freelance work opportunities
-- ``#community-help-wanted`` is a good place to share calls for participation on open-source projects
-- ``#???`` is a good place to announce non-Write the Docs events
-- ``#???`` is a good place to share a survey or calls for research interview subjects
+- ``#community-events`` (formerly known as `#calls-for-papers`) is a good place to announce non-Write the Docs events
```

Comment:
Let's make sure we update the channel description when we do this to mention the previously known as